### PR TITLE
Improve peripheral naming consistency #85

### PR DESCRIPTION
**Pull request by tlentali**
Hi @Fleker, I added the upper case name convention and changed the tests files with the correct upper case names. Is it ok for you ?

### CODE REVIEW

**Comment by tlentali** [comment]
I signed the Google Contributor License Agreements :) !

**Comment by tlentali** [comment]
@googlebot I signed it!

**Comment by tlentali** [comment]
I see that there is an issue on commit b2c. Unittest works in local on this commit, do you see what is wrong ?

File: src/codegen.py [hunk 1]
```diff
@@ -59,6 +59,11 @@ def generate_source_file(template, peripheral, template_extension, output_dir):
         if _DEBUG:
             print(peripheral_data)
 
+        output_file_name = peripheral_data["info"]["title"]
+
+        # Naming convention: upper case
+        peripheral_data["info"]["title"] = peripheral_data["info"]["title"].upper()
```
Comment (Fleker) [inline src/codegen.py]: I don't think we'll need to change the filename here. If we change the filename `peripherals/Mcp4725.yaml` and in that file change `info.title` to be the uppercase form, that should be sufficient./

Comment (tlentali) [reply src/codegen.py]: Ok I see, so I will change the `info.title` in `Bmp280.yaml` and `Mcp4725.yaml` then. Thanks @Fleker !

**Comment by Fleker** [comment]
Looks like unit tests are failing (`python3 -m unittest discover test`) as the sample files will need to be renamed as well.

**Comment by Fleker** [comment]
Unit tests still seem to be failing.

**Comment by tlentali** [comment]
Unittest seems ok now !

**Comment by Fleker** [comment]
Okay, the tests are passing. But can you also update the [cloudbuild.yaml](https://github.com/google/cyanobyte/blob/master/cloudbuild.yaml) with the new filepaths? That's the last step that is failing in the CI.

**Comment by tlentali** [comment]
Filepaths are correct now :)

**Comment by Fleker** [comment]
Thanks!

**Comment by tlentali** [comment]
Thank you very much for your help @Fleker !

**Pull request merged**